config/locales/en.yml

```diff
@@ -546,6 +546,15 @@ en:
         Click to assign or change the color of this priority.
         It can be used for highlighting work packages in the table.
 
+  reactions:
+    action_title: "React"
+    add_reaction: "Add reaction"
+    react_with: "React with %{reaction}"
+    and_user: "and %{user}"
+    and_n_others_singular: "and %{n} other"
+    and_n_others_plural: "and %{n} others"
```

Comment:
🟡 we should use pluralisation from I18n here like:

```yaml
and_others:
  one: and 1 other
  others: and %{count} others
```

example: see [here](https://github.com/opf/openproject/pull/17017/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R258-R261)